hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -58,6 +58,12 @@
   private static final String DEFAULT_ASSUME_DATE_PARTITIONING = "false";
   private static final String HOODIE_WRITE_STATUS_CLASS_PROP = "hoodie.writestatus.class";
   private static final String DEFAULT_HOODIE_WRITE_STATUS_CLASS = WriteStatus.class.getName();
+  private static final String HOODIE_COPYONWRITE_USE_TEMP_FOLDER = "hoodie.copyonwrite.use.temp.folder";
```

Comment:
mergehandle is also used in copyonwrite..

Can you rename the params to 
`hoodie.copyonwrite.use.temp.folder.for.create` , `hoodie.copyonwrite.use.temp.folder.for.merge` and rework the getter/setters